Makefile

```diff
@@ -12,7 +12,52 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+.PHONY: push-multiarch-% push-multiarch
+
 CMDS=csi-node-driver-registrar
 all: build
 
 include release-tools/build.make
+
+# Override registry, revision and Images
+REV=$(REV_CI)
+REGISTRY_NAME=$(REGISTRY_CI)
+IMAGE_NAME=$(REGISTRY_NAME)/$*
+
+# This target builds multiarch images using Moby BuildKit builder toolkit.
+# Docker Buildx is included in Docker 19.03 and needs DOCKER_CLI_EXPERIMENTAL enabled to run corresponding commands.
+# Currently amd, s390x and Windows manifest is pushed for canary, release branch and released tags.
+# Images generated from Prow build are pushed to staging area on gcr
+push-multiarch-%:
+	make BUILD_PLATFORMS="windows amd64 .exe"
+	set -ex; \
+	gcloud auth configure-docker; \
+	label_rev=v$$(echo $(REV) | cut -f3- -d 'v'); \
```

Comment:
Is this specific to the cloud build? Then put it into `cloudbuild.yaml` and explain what it does.

Reply:
Moved rev calculation to `cloudbuild.yaml`